pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -602,7 +602,7 @@ def plot_animated_field(
         ----------
         quantity : str
             Quantity to plot (e.g. ``"Mag_E"``).
```

Comment:
```suggestion
            Quantity to plot (for example, ``"Mag_E"``).
```
Avoid Latin phrases like e.g.